test/test_tagging.py

```diff
@@ -0,0 +1,110 @@
+import json
+from unittest.mock import (
+    PropertyMock,
+    patch,
+)
+
+from azul.deployment import (
+    populate_tags,
+)
+from azul.logging import (
+    configure_test_logging,
+)
+from azul_test_case import (
+    AzulUnitTestCase,
+)
+
+
+def setupModule():
+    configure_test_logging()
+
+
+class TestTagging(AzulUnitTestCase):
+
+    @patch('subprocess.run', new_callable=PropertyMock)
```

Comment:
Are you sure you need PropertyMock here? subprocess.run is a method. I think you want a MagicMock here or sth.